internal/cmds/iter.go

```diff
@@ -24,3 +27,10 @@ func (c XaddFieldValue) FieldValueIter(seq iter.Seq2[string, string]) XaddFieldV
 	}
 	return c
 }
+
+func (c ZaddScoreMember) ScoreMemberIter(seq iter.Seq2[float64, string]) ZaddScoreMember {
+	for score, member := range seq {
+		c.cs.s = append(c.cs.s, fmt.Sprintf("%f", score), member)
```

Comment:
```suggestion
		c.cs.s = append(c.cs.s, strconv.FormatFloat(score, 'f', -1, 64), member)
```

`strconv.FormatFloat` is much more performant than `fmt.Sprintf`